client/src/context/useAuthContext.tsx

```diff
@@ -25,7 +25,7 @@ export const AuthProvider: FunctionComponent = ({ children }): JSX.Element => {
   const updateLoginContext = useCallback(
     (data: AuthApiDataSuccess) => {
       setLoggedInUser(data.user);
-      history.push('/dashboard');
+      history.push('/profile');
```

Comment:
As per usual, is this (and the same change below) intended?

Reply:
Yes, it was. I added in the description of this PR that it would be reverted to its original value before merging, however it seems safer to leave it as it was and just add the instruction to change it for testing. Thoughts ❓

Reply:
In general, I recommend against leaving changes that are supposed to be modified/removed before merging as it can be easy to forget and also misleading for collaborators - hence a non-draft PR should only include relevant changes to the feature/module it implements.

That said, I'm not 100% familiar with why we do this change (seems to happen a lot), is it because we don't have the link to this page implemented yet? If so, it could be best to create a temporary link along with a `TODO` comment. 

Either way, my personal opinion is that whatever is the intended end result (e.g. redirect clients who log in to dashboard or profile) should be left in. You could also keep it modified and leave a `TODO` comment there.

Reply:
Thanks. I'll keep that in mind, for the following PR's